definitions/artifacts/azure-virtual-network.json

```diff
@@ -40,45 +40,20 @@
         "infrastructure": {
           "title": "Infrastructure configuration",
           "required": [
-            "vnet_id",
+            "id",
             "cidr",
-            "default_subnet_id",
-            "delegated_subnets"
+            "default_subnet_id"
           ],
           "type": "object",
           "properties": {
-            "vnet_id": {
+            "id": {
               "$ref": "../types/azure-resource-id.json"
             },
             "cidr": {
               "$ref": "../types/cidr.json"
             },
             "default_subnet_id": {
               "$ref": "../types/azure-resource-id.json"
-            },
```

Comment:
I don't think we need to do this now, but it might be nice to use a `data` resource in tf and send a list of subnets in case a downstream bundle needs them. Thoughts?

Reply:
It would be nice to have, but unfortunately I don't think we could rely on it.  Every time they added a new bundle that created the subnet they would unknowingly make the vnet artifact stale.

Since we pass forward the vnet id, any downstream bundle that needs to know about subnets could just do a data lookup then to get a live list of the subnets.